src/app/system/system-information/system-information.component.scss

```
@@ -0,0 +1,109 @@
+    margin-bottom: 16px;
+    font-size: 1.5rem;
+    font-weight: 400;
+  }
+
+  p {
+    line-height: 1.6;
+    margin-bottom: 12px;
+    color: var(--md-sys-color-on-surface-variant, #555);
+  }
+
+  ::ng-deep a {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)